### ISSUE TITLE
Can't compile packages

### ISSUE THREAD

**Issue by sfyfe31** [felixrieseberg/ember-electron#151]
I'm setting up ember and ember-electron from scratch on macOS:
```
npm install -g ember-cli
ember new EMBER_ELECTRON_BASE
cd EMBER_ELECTRON_BASE
ember install ember-electron
```

Then I've changed `locationType` in environment.js so my app will run in electron and on the web:
```
locationType: process.env.EMBER_CLI_ELECTRON ? 'hash' : 'auto',
```

That's it. So I would say a completely new vanilla Ember app, with ember-electron. But then when I run `ember electron:package` I get:
```
cleaning up...
Built project successfully. Stored in "./tmp/electron-build-tmp/dist".

Copying files into Electron Build folder
Copying electron.js
Copying package.json
Installing production dependencies into Electron Build Package
Recompiling native dependencies

Native compilation did not work:
Error: ENOENT: no such file or directory, scandir '/Users/scott/Sites/EMBER_ELECTRON_BASE/tmp/electron-build-tmp/node_modules'
```

Any ideas?

**Comment by kevingelion** [comment]
Looks like probably a duplicate of #131

**Comment by felixrieseberg** [comment]
This will be fixed with the release 2.0 - in the meantime, install *any* package as a dependency to your project (something tiny, maybe leftpad) and you should be good. In the beginning, I never accounted for people *not* having `node_modules` 😅
